### PR TITLE
Making the linter setup more specific in order to avoid stickler issues in Pull Requests

### PR DESCRIPTION
**Pull request by jofer86**
## Slightly more specific ESLINT configuration instructions
This PR solves an issue where the student might have set up the `.stickler.yml` file with the incorrect config file raising this stickler error on the PR due to having more than one config file
image
When the student has `eslint.config` file linked to their `.stickler.yml` file and not realizing this is the root of the problem having to resort to a solution like this one
image
With the solution described in the precedent lines, we rule out some guesswork that some students may have to go through by stating only one specific way to set up eslint and preventing them from future errors that may come from the improper set up of this tool.

The image currently not showing in this PR, in line 18 is the following:
image

I think the PR needs to be approved so that the image shows properly since it paths to the Microverse Repo.

### CODE REVIEW

File: react-redux/README.md [hunk 1]
```diff
@@ -15,10 +15,15 @@ Please do the following **steps in this order**:
 ### Set-up ESlint in your local env - it will help you to find style errors
 1. run `npm install eslint eslint-config-airbnb --save-dev` (not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm))
 2. run `npx eslint --init`
-3. copy [.eslintrc.json](./.eslintrc.json) to the root directory of your project
-4. run `npx eslint .`
-5. fix linter errors
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for ESlint](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+3. make sure you select the following options when prompted
+
+![screenshot](../assets/images/eslint_config.png)
```
Comment (bolah2009) [inline react-redux/README.md]: Could you verify that this file is showing? Seems to be broken, also note that the file is `eslint_config.PNG` not `eslint_config.png`.

```suggestion
![screenshot](../assets/images/eslint_config.PNG)
```

File: react-redux/README.md [hunk 2]
```diff
@@ -15,10 +15,15 @@ Please do the following **steps in this order**:
...
+
+4. copy the contents of [.eslintrc.json](./.eslintrc.json) to the newly generated `.eslintrc.json` overwritting the previous content.
+5. double check your `./src` folder for any extra unnecesary `.eslint` config files that might have been generated as this might cause an issue with stickler when you create your Pull Request later on
+6. run `npx eslint .`
+7. fix linter errors
+8. **IMPORTANT NOTE**: feel free to research [auto-correct options for ESlint](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
```
Comment (shloch) [inline react-redux/README.md]: nice shot

File: react-redux/README.md (hunk 1 above)
Comment (jofer86) [reply react-redux/README.md]: The path to the screenshot is going to the microverse repo that why it isn't showing, fixing it now

Comment (bolah2009) [reply react-redux/README.md]: Oh, I was viewing it from the commit that has the image: https://github.com/microverseinc/linters-config/blob/43f6e35499c9f4de04b16d0030fc9b6e35a66c71/react-redux/README.md

But it's all good now with the changes: https://github.com/microverseinc/linters-config/blob/ce946ae39b3a7f186e6533eb61e92c8ece3fc5bd/react-redux/README.md

**Comment by carloshdelreal** [comment]
before merging check out https://github.com/microverseinc/linters-config/pull/35

**Comment by nidalaa** [comment]
Thank you @jofer86 for this outstanding PR and even more (sic!) outstanding description!
It's a pleasure to merge it! 😻

**Pull request merged**